README.md

```diff
@@ -1,40 +1,40 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# Qibo site instructions
 
-It is an experimental repo, created to familiarize with `nextjs`, `javascript` and `react`.
+This is the Qibo's website repository.
+Here we describe its contents and show you how to run the site in developement mode, in order to collaborate with us.
 
-## How to run the site in developement mode
+
+### How to open the site in developement mode
 
 First, you need to clone this repo locally. For doing this, enter in the location in which you want the Qibo site folder and digit: 
 
 ```bash
-git clone https://github.com/qibogang/qibogang.github.io.git --recurse-submodules
+git clone https://github.com/qibogang/qibogang.github.io.git
```

Comment:
You can also assume people knows how to clone a repo ;)

Reply:
Thank you @AleCandido for the review. 
Now I have rm the useless `clone ...` and I have added the link to `virtualenv` installation instructions.